lib/Core/Errors.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Phpactor\WorseReflection\Core;
+
+use Phpactor\WorseReflection\Core\Inference\SymbolInformation;
+
+final class Errors implements \IteratorAggregate
+{
+    private $errors = [];
+
+    private function __construct($errors)
+    {
+        foreach ($errors as $item) {
+            $this->add($item);
+        }
+    }
+
+    public static function fromErrors(array $errors): Errors
+    {
+         return new self($errors);
+    }
+
+    public function getIterator()
+    {
+        return new \ArrayIterator($this->errors);
+    }
+
+    private function add(SymbolInformation $item)
+    {
+        $this->errors[] = $item;
+    }
+
+    public function __toString()
+    {
+        $lines = [];
+        /** @var SymbolInformation $symbolInformation */
+        foreach ($this->errors as $symbolInformation) {
+            $lines[] = sprintf(
+                '%s:%s %s',
+                $symbolInformation->symbol()->position()->start(),
+                $symbolInformation->symbol()->position()->end(),
```

Comment:
Would be ideal (maybe not essential, see what Ale/Syntastic support) to return the line number also